src/main/java/org/spongepowered/api/entity/player/Player.java

```diff
@@ -79,8 +82,11 @@
     Locale getLocale();
 
     /**
-     * Sends the plain text message(s) with the specified {@link ChatType} on the client.
-     * <p>Use {@link #sendMessage(ChatType, Message...)} for a formatted message.</p>
```

Comment:
Please fix your formatter from changing pre-existing code. You can format yor added code but do not format fix pre-existing code.